site/docs/v1/tech/installation-guide/cluster.adoc

```diff
@@ -46,6 +46,7 @@ When building a FusionAuth cluster, consider:
 * What level of security and network isolation do you need? Build the architecture that suits your needs. You can run FusionAuth in a private network and have all HTTP connections proceed through the load balancer, with SSH connections happening through a jump box, for example.
 * What version of FusionAuth will you be running? All nodes must be on the same version for correct functionality.
 * How will you manage link:/docs/v1/tech/reference/configuration/[FusionAuth configuration]? All nodes must have the same configuration or undetermined behavior will occur. Ensure that configuration changes are replicated to every node.
+* Will you use a custom password hashing plugin? If so, plan to distribute the plugin to every node.
```

Comment:
Or if using k8s, docker or the like, you can mount the plugin directory to share it amongst nodes. It looks like you already mention this below, so maybe ok.

Reply:
Fair, added.